common/src/main/java/io/netty/util/internal/logging/Slf4JLoggerFactory.java

```diff
@@ -40,31 +37,8 @@ public Slf4JLoggerFactory() {
 
     Slf4JLoggerFactory(boolean failIfNOP) {
         assert failIfNOP; // Should be always called with true.
-
-        // SFL4J writes it error messages to System.err. Capture them so that the user does not see such a message on
```

Comment:
Thanks... if @Scottmitch is fine with it I will just remove it. @Scottmitch WDYT ?